<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleSettingsDialog</class>
 <widget class="QWidget" name="GoogleSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QFormLayout" name="verticalLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="accountTextLabel">
     <property name="text">
      <string>Account:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="accountLabel">
     <property name="text">
      <string>&lt;b&gt;not configured&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="configureBtn">
     <property name="icon">
      <iconset theme="configure">
       <normaloff/>
      </iconset>
     </property>
     <property name="text">
      <string>Configure...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="enableRefresh">
     <property name="text">
      <string>Enable interval refresh</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="refreshLabel">
     <property name="text">
      <string>Refresh interval:</string>
     </property>
     <property name="buddy">
      <cstring>refreshSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="KPluralHandlingSpinBox" name="refreshSpinBox">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>720</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="eventsLimitLabel">
     <property name="text">
      <string>Fetch only events since:</string>
     </property>
     <property name="buddy">
      <cstring>eventsLimitCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="KDateComboBox" name="eventsLimitCombo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="calendarsLabel">
     <property name="text">
      <string>Enabled Calendars:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeft|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>calendarsList</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QListWidget" name="calendarsList"/>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reloadCalendarsBtn">
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="taskListsLabel">
     <property name="text">
      <string>Enabled Tasklists:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeft|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>taskListsList</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QListWidget" name="taskListsList"/>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reloadTaskListsBtn">
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>enableRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>refreshSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
